### PR TITLE
Fix broken documentation links

### PR DESCRIPTION
**Pull request by anjbur**

### CODE REVIEW

**Review by tcNickolas — COMMENTED** [review]
Standard/README.md shows as binary (weird!), but looking at it in https://github.com/microsoft/QuantumLibraries/blob/2c2c786fd8a75eb0295da2adbd8204cf54e54e23/Standard/README.md, it says "Q# sources used to implement the canon portion" but points to [chemistry](https://docs.microsoft.com/azure/quantum/user-guide/libraries/chemistry) - this doesn't look right?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Welcome, and thank you for your interest in contributing to the Quantum Development Kit!
 
 There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
-For more details on how to contribute to the samples or to the rest of the Quantum Development Kit, please see the [contribution guide](https://docs.microsoft.com/quantum/contributing/).
+For more details on how to contribute to the samples or to the rest of the Quantum Development Kit, please see the [contribution guide](https://docs.microsoft.com/azure/quantum/contributing-overview/).
```
Comment (cgranade) [inline CONTRIBUTING.md:6]: That's weird, I wonder why this was moved?

**Review by cgranade — COMMENTED** [review]
> Standard/README.md shows as binary (weird!),

That can often be an artifact of some tools not correctly parsing byte order marks in UTF-8 encoded files (UTF-8 with BOM is a non-standard MS extension to the Unicode spec). As a result, we may be able to fix by removing the first two bytes. Using `Set-Content -Encoding Utf8NoBOM` to normalize the README confirms that this is likely the case.

File: CONTRIBUTING.md (hunk 1 above)
Comment (anjbur) [reply CONTRIBUTING.md:6]: Unclear, it looks like it should point to the same place though! It links to the main page in the "Contributing to the QDK" documentation section

**Comment by anjbur** [comment]
The link is fixed. The README still shows as a binary in the "Files Changed" summary even though it correctly looks like a text file if you look at just the most recent commit. @cgranade is that something that needs to be fixed? Do you know why it would show differently in the per-commit view versus the overall view?

**Review by tcNickolas — APPROVED** [review]

**Pull request merged**